### ISSUE TITLE
reproject grid data for rasterio / gdal

### ISSUE THREAD

**Issue by vodp** [fmaussion/salem#230]
I find salem super helpful to deal with .nc files. My tasks however somtimes involves with reprojecting nc gridded data to be used with rasterio and visualizing on top of Google Maps. I find that merely setting crs to epsg:4326 is not correct to show nc data. For example in the following data of global soil moisture product, overlaying on top of Esri map gives incorrect results:
![image](https://user-images.githubusercontent.com/7162458/223399891-85645a6f-9924-4c3f-a576-a6b16df51d95.png)

but with the internal quick_map of salem, the country boundaries are correctly visualized:

![image](https://user-images.githubusercontent.com/7162458/223399998-3c6be663-f5f0-4e0d-a6e7-dd9cc7ea2c8e.png)

I am wondering is there a mechanism to help project the grid data so that it can be conveniently used with Transform matrix, and Crs as like rasterio or Gdal ?

**Comment by fmaussion** [comment]
I'm sorry I can't help you there. You should have a look at https://corteva.github.io/rioxarray though, this is much better than salem!

**Comment by fmaussion** [comment]
Yes Sorry Salem does not support 0-360 grids. I recommend to shift your dataset to -180 > 180 (using xarray `.roll`), or use cartopy for plotting. If Salem can read it, your dataset is probably a straightforward plate carree projection
